src/repos/ArticleRepo.ts

```diff
@@ -36,20 +38,35 @@ const getArticlesAfterDate = async (since: string, limit = Constants.DEFAULT_LIM
   );
 };
 
+/** A function to compare the trendiness of articles.
+ * 
+ * Trendiness is computed by taking the number of total shoutouts an article
+ * has received and dividing it by the number of days since its been published.
+ *
+ */
+export const compareTrendiness = (a1: Article, a2: Article) => {
+  const presentDate = new Date().getTime();
+  const a1Score = a1.shoutouts / (presentDate - a1.date.getTime());
+  const a2Score = a2.shoutouts / (presentDate - a2.date.getTime());
+  return a2Score - a1Score;
+};
+
 /**
  * Computes and returns the trending articles in the database.
  *
  * @function
  * @param {number} limit - number of articles to retrieve.
  * @param {string} since - retrieve articles after this date.
  */
-const getTrendingArticles = async (since: string, limit = Constants.DEFAULT_LIMIT): Promise<Article[]> => {
-  const trendingArticles = await ArticleModel.find({
+const getTrendingArticles = async (
+  since: string,
+  limit = Constants.DEFAULT_LIMIT,
+): Promise<Article[]> => {
+  const articlesSinceDate = await ArticleModel.find({
     date: { $gte: new Date(new Date(since).setHours(0, 0, 0)) },
-  })
-    .sort({ trendiness: 'desc' })
-    .exec();
-  return trendingArticles.slice(0, limit);
+  }).exec();
+
+  return articlesSinceDate.sort(compareTrendiness).slice(0, limit);
```

Comment:
is this sorted from most trendy to least? or least trendy to most? probably helpful to comment that